ios/ContactPicker.m

```diff
@@ -43,6 +43,11 @@ - (instancetype)init {
     self.resolve = resolve;
     self.reject = reject;
     self.pickerController.peoplePickerDelegate = self;
+    
+    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
This block needs to go in the two places that currently call presentViewController instead of up here, so it still checks for the permissions. Also need to do the same sort of block around dismissViewControllerAnimated inside of peoplePickerNavigationController.

Reply:
updated, please let me know if this is correct. I'm not entirely knowledgeable in this code